testdrive/github.go

```diff
@@ -56,7 +56,6 @@ func (g *Client) CheckForkSuccess(ownerName string, forkRepoName string) bool {
 // CreateWebhook creates a GitHub webhook to send requests to our local ngrok.
 func (g *Client) CreateWebhook(ownerName string, repoName string, hookURL string) error {
 	atlantisHook := &github.Hook{
-		Name:   github.String("web"),
```

Comment:
why is this removed?

Reply:
See here:

https://github.com/google/go-github/pull/1032